docs/src/modules/components/Ad.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Tooltip from '@material-ui/core/Tooltip';
+import withWidth from '@material-ui/core/withWidth';
+import CodeFund from 'docs/src/modules/components/CodeFund';
+import CarbonAds from 'docs/src/modules/components/CarbonAds';
+
+const styles = theme => ({
+  root: {
+    position: 'relative',
```

Comment:
What is this for?

Reply:
It's straight out of the current `CodeFund.js` implementation...

Reply:
Looks like it's for the `position: absolute` of the info section to position against.